src/jarabe/journal/journaltoolbox.py

```diff
@@ -727,11 +724,20 @@ def __init__(self):
             menu_box.append_item(button)
 
     def __sort_type_changed_cb(self, widget, property_, icon_name):
+        if(self._property == property_):
```

Comment:
Please run flake8 on this, but I don't think that it is accepted to put the parentheses in the if statement like C.